week.go

```diff
@@ -6,6 +6,7 @@ import (
 	"time"
 
 	"github.com/pkg/errors"
+	"math"
```

Comment:
I usually prefer to organize imports  such that standard library imports come first, then an empty line, and at the end non standard library imports. Using `goimports` can help to keep imports tidy.